diff --git a/node_modules/capacitor-google-nearby-messages/dist/esm/definitions.d.ts b/node_modules/capacitor-google-nearby-messages/dist/esm/definitions.d.ts
index ac9bb7e..be16fa4 100644
--- a/node_modules/capacitor-google-nearby-messages/dist/esm/definitions.d.ts
+++ b/node_modules/capacitor-google-nearby-messages/dist/esm/definitions.d.ts
@@ -93,8 +93,8 @@ export interface GoogleNearbyMessagesPlugin {
     addListener(eventName: 'onPermissionChanged', listenerFunc: (permissionGranted: boolean) => void): PluginListenerHandle;
     addListener(eventName: 'onBleSignalChanged', listenerFunc: (message: Message, bleSignal: BleSignal) => void): PluginListenerHandle;
     addListener(eventName: 'onDistanceChanged', listenerFunc: (message: Message, distance: Distance) => void): PluginListenerHandle;
-    addListener(eventName: 'onFound', listenerFunc: (message: Message) => void): PluginListenerHandle;
-    addListener(eventName: 'onLost', listenerFunc: (message: Message) => void): PluginListenerHandle;
+    addListener(eventName: 'onFound', listenerFunc: (data: {message: Message}) => void): PluginListenerHandle;
+    addListener(eventName: 'onLost', listenerFunc: (data: {message: Message}) => void): PluginListenerHandle;
     addListener(eventName: 'onPublishExpired', listenerFunc: (uuid: UUID) => void): PluginListenerHandle;
     addListener(eventName: 'onSubscribeExpired', listenerFunc: (uuid: UUID) => void): PluginListenerHandle;
 }
